### ISSUE TITLE
Remove self referencing tests in Html.roc

### ISSUE THREAD

**Issue by normanjaeckel** [Hasnep/roc-html#5]
These four lines of testing code contain self referencing function calls.

https://github.com/Hasnep/roc-html/blob/081db01bd28a23a81657d6205d932192d9d38fc0/src/Html.roc#L156-L157
https://github.com/Hasnep/roc-html/blob/081db01bd28a23a81657d6205d932192d9d38fc0/src/Html.roc#L174-L175

This only works if I import the Html package directly in the main.roc of my app. But I want to import the package only in an interface file. This causes `roc check` to fail because from my main.roc roc goes to the packages record and from there to your (btw really good!) package and then ...
```
── MODULE NOT IMPORTED in ...QTpMYIZkigkDa2rfTc92wt-P_lsa76JVXb8Qb3ms/Html.roc ─

The `Html` module is not imported:

156│      textNode = Html.text "<script>alert('hi')</script>"
                     ^^^^^^^^^

Did you mean to import it?
```

Thanks for fixing this or even any hint how can I fix it myself.

**Comment by Hasnep** [comment]
Hi, thanks for reporting this bug, I might have some time later this week to take a look at fixing it :) Also I'm not going to take any credit for writing this package, it was >95% written by Brian Carroll 😅

**Comment by Hasnep** [comment]
Hi again @normanjaeckel, I've released a new version of this package (v0.2.1) that removes the references to the `Html` interface from within itself, could you try replacing the URL to roc-html with this one: `https://github.com/Hasnep/roc-html/releases/download/v0.2.1/gvFCxQTb3ytGwm7RQ87BVDMHzo7MNIM2uqY4GBDSP7M.tar.br` and then try running `roc check` again? Let me know if that fixes the error or not :)

**Comment by normanjaeckel** [comment]
Works fine. Thank you very much. ;-)
